### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by ngfuong** [AbdelrhmanNile/UnderTaker141#5]
Downloading Blasphemous 2 resulted in the following error
```
Traceback (most recent call last):                                                                                                                                                                                                           
   File "/tmp/.mount_UnderTsv4Mio/opt/src/app.py", line 19, in <module>                                                                                                                                                                       
     UnderTaker141().run()                                                                                                                                                                                                                    
   File "/tmp/.mount_UnderTsv4Mio/opt/python3.11/lib/python3.11/site-packages/kivy/app.py", line 956, in run                                                                                                                                  
     runTouchApp()                                                                                                                                                                                                                            
   File "/tmp/.mount_UnderTsv4Mio/opt/python3.11/lib/python3.11/site-packages/kivy/base.py", line 574, in runTouchApp                                                                                                                         
     EventLoop.mainloop()                                                                                                                                                                                                                     
   File "/tmp/.mount_UnderTsv4Mio/opt/python3.11/lib/python3.11/site-packages/kivy/base.py", line 339, in mainloop                                                                                                                            
     self.idle()                                                                                                                                                                                                                              
   File "/tmp/.mount_UnderTsv4Mio/opt/python3.11/lib/python3.11/site-packages/kivy/base.py", line 383, in idle         
     self.dispatch_input()                                 
   File "/tmp/.mount_UnderTsv4Mio/opt/python3.11/lib/python3.11/site-packages/kivy/base.py", line 334, in dispatch_input
     post_dispatch_input(*pop(0))                          
   File "/tmp/.mount_UnderTsv4Mio/opt/python3.11/lib/python3.11/site-packages/kivy/base.py", line 263, in post_dispatch_input
     listener.dispatch('on_motion', etype, me)                                                                         
   File "kivy/_event.pyx", line 731, in kivy._event.EventDispatcher.dispatch                                           
   File "/tmp/.mount_UnderTsv4Mio/opt/python3.11/lib/python3.11/site-packages/kivy/core/window/__init__.py", line 1691, in on_motion
     self.dispatch('on_touch_down', me)                                                                                
   File "kivy/_event.pyx", line 731, in kivy._event.EventDispatcher.dispatch                                           
   File "/tmp/.mount_UnderTsv4Mio/opt/python3.11/lib/python3.11/site-packages/kivy/core/window/__init__.py", line 1708, in on_touch_down
     if w.dispatch('on_touch_down', touch):                                                                            
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                             
   File "kivy/_event.pyx", line 731, in kivy._event.EventDispatcher.dispatch                                           
   File "/tmp/.mount_UnderTsv4Mio/opt/python3.11/lib/python3.11/site-packages/kivy/uix/modalview.py", line 266, in on_touch_down
     super().on_touch_down(touch)                          
   File "/tmp/.mount_UnderTsv4Mio/opt/python3.11/lib/python3.11/site-packages/kivy/uix/widget.py", line 589, in on_touch_down
     if child.dispatch('on_touch_down', touch):                                                                        
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                         
   File "kivy/_event.pyx", line 731, in kivy._event.EventDispatcher.dispatch                                           
   File "/tmp/.mount_UnderTsv4Mio/opt/python3.11/lib/python3.11/site-packages/kivymd/uix/behaviors/ripple_behavior.py", line 352, in on_touch_down
     super().on_touch_down(touch)                          
   File "/tmp/.mount_UnderTsv4Mio/opt/python3.11/lib/python3.11/site-packages/kivy/uix/behaviors/button.py", line 138, in on_touch_down
     if super(ButtonBehavior, self).on_touch_down(touch):                                                              
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                               
   File "/tmp/.mount_UnderTsv4Mio/opt/python3.11/lib/python3.11/site-packages/kivy/uix/widget.py", line 589, in on_touch_down
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   File "/tmp/.mount_UnderTsv4Mio/opt/python3.11/lib/python3.11/site-packages/kivy/uix/widget.py", line 589, in on_touch_down
     if child.dispatch('on_touch_down', touch):
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   File "kivy/_event.pyx", line 731, in kivy._event.EventDispatcher.dispatch
   File "/tmp/.mount_UnderTsv4Mio/opt/python3.11/lib/python3.11/site-packages/kivy/uix/widget.py", line 589, in on_touch_down
     if child.dispatch('on_touch_down', touch):
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   File "kivy/_event.pyx", line 731, in kivy._event.EventDispatcher.dispatch
   File "/tmp/.mount_UnderTsv4Mio/opt/python3.11/lib/python3.11/site-packages/kivy/uix/widget.py", line 589, in on_touch_down
     if child.dispatch('on_touch_down', touch):
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   File "kivy/_event.pyx", line 731, in kivy._event.EventDispatcher.dispatch
   File "/tmp/.mount_UnderTsv4Mio/opt/python3.11/lib/python3.11/site-packages/kivymd/uix/button/button.py", line 1173, in on_touch_down
     return super().on_touch_down(touch)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   File "/tmp/.mount_UnderTsv4Mio/opt/python3.11/lib/python3.11/site-packages/kivymd/uix/behaviors/ripple_behavior.py", line 352, in on_touch_down
     super().on_touch_down(touch)
   File "/tmp/.mount_UnderTsv4Mio/opt/python3.11/lib/python3.11/site-packages/kivy/uix/behaviors/button.py", line 151, in on_touch_down
     self.dispatch('on_press')
   File "kivy/_event.pyx", line 727, in kivy._event.EventDispatcher.dispatch
   File "kivy/_event.pyx", line 1307, in kivy._event.EventObservers.dispatch
   File "kivy/_event.pyx", line 1231, in kivy._event.EventObservers._dispatch
   File "/tmp/.mount_UnderTsv4Mio/opt/src/widgets/game/game.py", line 73, in download
     library.layout.add_widget(GameLibraryCard(game_torrent=qbt.get_torrent(self.magnet)))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   File "/tmp/.mount_UnderTsv4Mio/opt/src/widgets/game/game.py", line 84, in __init__
     self.game_obj = db.get_game(self.game_name)[0]
                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^
 IndexError: list index out of range
                                                                                                                                                                                                 
```

**Comment by AbdelrhmanNile** [comment]
does it happen immediately after you click download?

**Comment by AbdelrhmanNile** [comment]
fixed in v1.1.2
please download the [latest build](https://github.com/AbdelrhmanNile/UnderTaker141/releases/tag/latest)
